cli/script/command-executor.ts

```diff
@@ -1290,4 +1297,10 @@ function whoami(command: cli.ICommand): Promise<void> {
         .then((account): void => {
             log(`${account.email} (${account.linkedProviders.join(", ")})`);
         });
-}
\ No newline at end of file
+}
+
+function getProxy(proxy: string, noProxy?: boolean): string {
```

Comment:
em, i think optional or not are both ok, since currently all places called `getProxy` pass two args. but for future usage, optional is more proper